primitive/redjubjub/Cargo.toml

```diff
@@ -18,9 +18,13 @@ jub-jub = { path = "../jubjub", default-features = false }
 bls-12-381 = { path = "../bls12_381", default-features = false }
 blake2b_simd = { version = "1", default-features = false }
 rand_core = { version="0.6.4", default-features = false }
+zero-plonk = { path = "../plonk", default-features = false }
```

Comment:
redjubjub is pure signing algorithm implementation
circuit implementation should be under zk_rollup